access/management/commands/grade.py

```diff
@@ -50,8 +45,3 @@ def handle(self, *args, **options):
 
             else:
                 raise CommandError("Submit file not found: %s" % (name))
-
-        # Run actions.
-        r = runactions(course, exercise, sdir)
```

Comment:
`runactions` is part of the chroot sandbox, but when you just remove the code here, clearly this `grade` command no longer grades anything. This command should now start the container grader.

Reply:
This `grade` command is supposed to grade a submission, so it must start the grader program. Graders can't be run in the old chroot sandbox anymore, so this command must start the grading container by calling `accept.types.stdasync._acceptSubmission`.

Reply:
Note that when `runactions` is called directly here, it runs the exercise grader synchronously. This grade command was intended for testing the grader, not for real production use. We can't really run container graders synchronously, so there might not be a way to receive the output of the container (feedback and points) here. Without the output, running the grader is rather pointless, so the whole command might become obsolete.